src/components/EventDetails/EventDetails.js

```diff
@@ -18,862 +18,1089 @@
-          }}
-          scrollEventThrottle={16}
-          ref={getRef}
-          scrollToOverflowEnabled={false}
-          bounces={false}
+      <React.Fragment>
+        <SlideModal
+          isVisible={isVisible}
+          onModalHide={onClose}
+          noClose
+          hideHeader
+          sideMargins={spacing.large}
         >
-          {this.renderEventBody(eventType)}
-        </ScrollView>
-        <SafeAreaView forceInset={{ top: 'never', bottom: 'always' }}>
-          {this.renderEventButtons(eventType, eventStatus)}
-        </SafeAreaView>
-      </ContentWrapper>
+          <Wrapper forceInset={{ top: 'never', bottom: 'always' }}>
+            <BaseText tiny secondary>{eventTime}</BaseText>
+            <Spacing h={10} />
+            <BaseText medium >{name}</BaseText>
+            <Spacing h={20} />
+            {this.renderImage(eventData)}
+            <Spacing h={20} />
+            {settleEventData ? this.renderSettle(settleEventData) : (
+              <React.Fragment>
+                <ActionWrapper>
+                  {!!actionTitle && <MediumText large color={actionColor}>{actionTitle}</MediumText>}
+                  {customActionTitle}
+                  {!!actionIcon && <ActionIcon name={actionIcon} />}
+                </ActionWrapper>
+                {actionSubtitle ? (
+                  <React.Fragment>
+                    <Spacing h={4} />
+                    <BaseText regular secondary>{actionSubtitle}</BaseText>
+                    <Spacing h={24} />
+                  </React.Fragment>
+                ) : (
+                  <Spacing h={32} />
+                )}
+                {!!fee && (
+                  <React.Fragment>
+                    <BaseText regular secondary>{fee}</BaseText>
+                    <Spacing h={32} />
+                  </React.Fragment>
+              )}
+              </React.Fragment>
+            )}
+            <ButtonsContainer>
+              {buttons.map(buttonProps => (
+                <React.Fragment key={buttonProps.title} >
+                  <Button regularText {...buttonProps} />
+                  <Spacing h={4} />
+                </React.Fragment>
+              ))}
+            </ButtonsContainer>
+          </Wrapper>
+        </SlideModal>
+        <ReceiveModal
+          isVisible={isReceiveModalVisible}
+          address={activeAccountAddress}
+          onModalHide={() => this.setState({ isReceiveModalVisible: false })}
+        />
+        <SWActivationModal
+          isVisible={SWActivationModalVisible}
+          onClose={() => this.setState({ SWActivationModalVisible: false })}
+          navigation={navigation}
+        />
+      </React.Fragment>
     );
   }
 }
 
 const mapStateToProps = ({
+  rates: { data: rates },
+  appSettings: { data: { baseFiatCurrency } },
   contacts: { data: contacts, contactsSmartAddresses: { addresses: contactsSmartAddresses } },
-  txNotes: { data: txNotes },
```

Comment:
Why we are no longer showing transaction notes?

Reply:
There is no place for transaction notes on design.